n3fit/src/n3fit/scripts/evolven3fit_new.py

```diff
@@ -152,7 +154,7 @@ def main():
         tcard, opcard = eko_utils.construct_eko_cards(
             args.theoryID, args.q_fin, args.q_points, x_grid, op_card_info, theory_card_info
         )
-        _ = eko_utils.construct_eko_for_fit(tcard, opcard, args.dump)
+        _ = runner.solve(tcard, opcard, args.dump)
```

Comment:
In the current version of EKO, `runner.solve` has no return value (because it was useless with the new output, as proven here)
```suggestion
        runner.solve(tcard, opcard, args.dump)
```